akka-http-core/src/main/scala/akka/http/impl/settings/ClientConnectionSettingsImpl.scala

```diff
@@ -24,7 +24,8 @@ private[akka] final case class ClientConnectionSettingsImpl(
   requestHeaderSizeHint: Int,
   websocketRandomFactory: () ⇒ Random,
   socketOptions: immutable.Seq[SocketOption],
-  parserSettings: ParserSettings)
+  parserSettings: ParserSettings,
+  redirectSettings: akka.http.scaladsl.settings.ClientAutoRedirectSettings)
```

Comment:
Speaking of binary compatibility, I'm currently struggling with another class, namely `akka.http.impl.settings.ClientConnectionSettingsImpl`. I've been able to create a constructor without the newly added parameter, however `mimaReportBinaryIssues` also complains about other case classes methods like `copy` and `apply`. AFAIK those are compiler generated and I can't override them. Or can I? And should I?